1.1 1.5 feature list ideas.md

```diff
@@ -32,23 +32,23 @@ Probably later:
 * Implement a 'I need emergency help button' #60
 * Make a new 'geofencing' on device app, which messages when a node enters/leaves a place.  Could allow **very** long battery life for animal tracking.
 * Store and forward messaging - If the mesh gets fragmented or a node goes out of range for x-period of time, eventually deliver the messages to the node that has rejoined. Especially useful for the hiking use case because someone may be over a ridge and have lost the Line of Sight.
-* TTGO v2.0 2.1 navigation buttons soldered on the board(ability to customize button gpios via preferences)
+* TTGO v2.0 2.1 navigation buttons soldered on the board(ability to customize button GPIOs via preferences)
 * Extend the portduino simulator to run whole meshes of nodes in simulation.
 * Enable a specific node in the mesh to log all position packets to a file (either in the App or SD card on device) to allow post activity route analysis of each node 
 * Buzzer support: Beeps for message received, node join, low battery etc
 * Let user subscribe to multiple channels at once
 * show compass heading in degrees as well as arrow. Makes it easier to track the  true direction of a person in distress.
 * add text-to-speech option in the app, for hands/screen free monitoring (while busy with stuff) and warnings of system events (new node discovered, low battery etc...)
 * add speech-to-text to the app, now it can be used almost as a normal phone.
-* External buttons - Have firmware recognise several exposed GPIO pins for 'scroll', 'event', etc. so that external mom.spst (reed sw, mercury sw. etc) can trigger these events. At present, we must expose device to the elements, or manufacture weatherproof actuators (difficult).
-* Let app distribute APKs of itself over bluetooth or wifi: per [this forum post](https://meshtastic.discourse.group/t/suggestion-ability-to-transfer-app-apk-to-another-android-device-via-bt-or-other-non-network-dependent-method/711). let the device distribute the app or have a wifi web client for basic use for any wifi enabled device.
+* External buttons - Have firmware recognize several exposed GPIO pins for 'scroll', 'event', etc. so that external mom.spst (reed sw, mercury sw. etc) can trigger these events. At present, we must expose device to the elements, or manufacture weatherproof actuators (difficult).
```

Comment:
You changed it to the wrong spelling! 😄

Reply:
These two markdown files in the repo root are going to get removed, they were placed here temporarily so we could clean up the device repo.

Reply:
Was just poking fun as it was changed it from English to American English spellling 😄